src/pipecat/audio/vad/vad_analyzer.py

```diff
@@ -8,7 +8,7 @@
 from enum import Enum
 
 from loguru import logger
-from pydantic.main import BaseModel
+from pydantic import BaseModel
```

Comment:
sphinx was complaining about pydantic.main. Is there any issue with this change? I've made it in a few places.